src/cltk/nlp.py

```diff
@@ -58,7 +58,12 @@ class NLP:
     process_objects = dict()
     process_lock = Lock()
 
-    def __init__(self, language: str, custom_pipeline: Pipeline = None) -> None:
+    def __init__(
```

Comment:
Did `black` format this? Please run `make format` if you have not already.

Reply:
oops, fixed, I believe

Reply:
If you don't, I will when running locally. Resolving